src/js/dropdown-menu/specs/dropdown-menu-view-spec.js

```diff
@@ -256,21 +257,33 @@ define([
                     closeOnKeypressTest(constants.keyCodes.space);
                 });
 
-                it('should return to the button after pressing down arrow key while at bottom of dropdown menu', function() {
-                    focusTrapDown(constants.keyCodes.down, 4);
-                });
+                it(
```

Comment:
Yeah, 120 is our max and these were too long. We can fix it by just shortening the message as well.